docusaurus/docs/Android/03-ui/05-components/05-message-list.mdx

```diff
@@ -461,4 +461,19 @@ If your predicate doesn't show avatars for your own messages (this is the defaul
 
 ```xml
 streamUiMessageEndMargin="0dp"
-```
\ No newline at end of file
+```
+## Configure giphy
+It is possible to configure the gifs size. Prior of version 4.23.0 gifs are autosizable and will resize thenselves accordingly with the size of the image they load. It is possible to use constant size gifs using `streamUiMediaAttachmentConstantSize` and set the desired height using `streamUiMediaAttachmentGiphyHeight`.
```

Comment:
There's a minor typo 'thenselves' instad of 'themselves'.

Also are attributes 'streamUiMediaAttachmentGiphyHeight' (should be `streamUiGiphyMediaAttachmentGiphyMaxHeight`) and `streamUiMediaAttachmentConstantSize` relevant anymore?

`streamUiMediaAttachmentConstantSize` seems to be missing from attributes and without it `streamUiGiphyMediaAttachmentGiphyMaxHeight` should be irrelevant. IIRC we did away with using both of them, please correct me if I'm wrong.